docs/CMakeLists.txt

```diff
@@ -1 +1,14 @@
-add_subdirectory(sphinx)
\ No newline at end of file
+add_custom_target(docs)
+
+if (SPHINX_FOUND)
+  add_subdirectory(sphinx)
+endif()
+
+if (DOXYGEN_FOUND)
+  add_subdirectory(doxygen)
+endif()
+
+if ( NOT SPHINX_FOUND AND NOT DOXGEN_FOUND)
+  message(WARNING "RAJA_ENABLE_DOCUMENTATION=On, but Sphinx or Doxygen not found. \
```

Comment:
Should this be broken out into two checks, one for sphinx and one for doxygen? This would make it easier for users if only one is found.